packages/otelbin/src/app/og/[id]/route.tsx

```diff
@@ -0,0 +1,64 @@
+// SPDX-FileCopyrightText: 2023 Dash0 Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { ImageResponse, type NextRequest } from "next/server";
+import { calcNodes } from "~/components/react-flow/useClientNodes";
+import ParentsNode from "../ParentsNode";
+import { Redis } from "@upstash/redis/nodejs";
+import { getShortLinkPersistenceKey } from "~/lib/shortLink";
+import { calcScale, parseUrlFragment } from "~/lib/metadataUtils";
+import type { IConfig } from "~/components/react-flow/dataType";
+
+export const runtime = "edge";
+
+const redis = Redis.fromEnv();
+const edgeWidth = 80;
+
+export async function GET(request: NextRequest) {
+	const shortLinkId = request.nextUrl.searchParams.get("id") ?? "";
+	const fullLink = (await redis.get<string>(getShortLinkPersistenceKey(shortLinkId))) ?? "";
```

Comment:
**Question:** Same as down below, it looks like this is missing error handling for the case that the ID doesn't resolve to a config.